src/zenml/zen_stores/schemas/artifact_schemas.py

```diff
@@ -351,7 +352,9 @@ def to_model(
                 artifact_store_id=self.artifact_store_id,
                 producer_step_run_id=producer_step_run_id,
                 visualizations=[v.to_model() for v in self.visualizations],
-                run_metadata={m.key: m.to_model() for m in self.run_metadata},
+                run_metadata={
+                    m.key: json.loads(m.value) for m in self.run_metadata
```

Comment:
Do we maybe need the pydantic json encoder here so this works with `UUID` and `datetime`? Just in case this ends up in some of the metadata dictionaries?

Reply:
Same applies to all the other schemas as well